protocol/messenger_communities.go

```diff
@@ -1704,6 +1704,34 @@ func (m *Messenger) ShareCommunity(request *requests.ShareCommunity) (*Messenger
 	return response, nil
 }
 
+func (m *Messenger) ParseCommuntyShareURL(url string, byID bool) (*MessengerResponse, error) {
+	var community *communities.Community
+	if byID {
+		// TODO: decompress community key
+		communityId := types.HexBytes(url)
+		community, err := m.GetCommunityByID(communityId)
```

Comment:
hmmm... This will only work for "known" communities.
But I suppose in most cases user will user shared url for unknown community. Probably we should start fetching community details here, os smth like that.

Same for profile below. There it will use `BuildContact` and return smth for unknown contact, but probably still fetching name/image is a good idea.

Reply:
There are 2 groups: 'c' which only provides community ID and `c#` with community data in protobuf. Here i'm waiting for conversion functions from Boris